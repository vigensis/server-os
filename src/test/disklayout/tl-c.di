SCSI	disk0	INTEL	DCS3700	800000000000	no	yes
SCSI	disk1	INTEL	DCS3700	800000000000	no	yes
SCSI	disk2	INTEL	DCS3700	800000000000	no	yes
SCSI	disk3	INTEL	DCS3700	800000000000	no	yes
SCSI	disk4	INTEL	DCS3700	800000000000	no	yes
SCSI	disk5	INTEL	DCS3700	800000000000	no	yes
SCSI	disk6	INTEL	DCS3700	800000000000	no	yes
SCSI	disk7	INTEL	DCS3700	800000000000	no	yes
SCSI	disk8	INTEL	DCS3700	800000000000	no	yes
USB	disk9	Kingston	DT Rubber 3.0	15737028608	yes	no
