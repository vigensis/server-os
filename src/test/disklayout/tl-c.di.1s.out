{
	"spares": [
		{
			"name": "disk8",
			"vid": "INTEL",
			"pid": "DCS3700",
			"size": "800000000000",
			"solid_state": true
		}
	],
	"vdevs": [
		{
			"type": "raidz2",
			"devices": [
				{
					"name": "disk0",
					"vid": "INTEL",
					"pid": "DCS3700",
					"size": "800000000000",
					"solid_state": true
				},
				{
					"name": "disk1",
					"vid": "INTEL",
					"pid": "DCS3700",
					"size": "800000000000",
					"solid_state": true
				},
				{
					"name": "disk2",
					"vid": "INTEL",
					"pid": "DCS3700",
					"size": "800000000000",
					"solid_state": true
				},
				{
					"name": "disk3",
					"vid": "INTEL",
					"pid": "DCS3700",
					"size": "800000000000",
					"solid_state": true
				},
				{
					"name": "disk4",
					"vid": "INTEL",
					"pid": "DCS3700",
					"size": "800000000000",
					"solid_state": true
				},
				{
					"name": "disk5",
					"vid": "INTEL",
					"pid": "DCS3700",
					"size": "800000000000",
					"solid_state": true
				},
				{
					"name": "disk6",
					"vid": "INTEL",
					"pid": "DCS3700",
					"size": "800000000000",
					"solid_state": true
				},
				{
					"name": "disk7",
					"vid": "INTEL",
					"pid": "DCS3700",
					"size": "800000000000",
					"solid_state": true
				}
			]
		}
	],
	"capacity": 4800000000000
}
